custom_student_verification/signals.py

```diff
@@ -18,6 +18,8 @@ def on_student_verification_request_saved(sender, instance, **kwargs):  # pylint
     """
     if instance.status == 'ACCEPTED':
         approve_user(instance.user, instance.reason)
+    elif instance.status == 'REJECTED':
+        reject_user_verification(instance.user, instance.reason)
```

Comment:
@arjunsinghy96 Can we use a similar naming for both functions? `approve_user` and `reject_user_verification`?